### DISCUSSION TITLE
2.17.8 and 2.17.9 software start very slowly, while 2.17.7 is normal

### DISCUSSION THREAD

**Discussion by SutoLiu** [Mygod/VPNHotspot#573] [General]
![vpnhotspot-2089535170184530619.zip](https://github.com/Mygod/VPNHotspot/files/15211718/vpnhotspot-2089535170184530619.zip)

@Mygod

**Comment by SutoLiu** [comment]
The issue of slow startup was caused by the Maven dependency "com.google.firebase:firebase-crashlytics". The problem was resolved by rolling back to version 18.6.3. @Mygod

**Reply by Mygod** [reply]
I cannot reproduce this on my side. Maybe report to Firebase then? https://github.com/firebase/firebase-android-sdk/issues

**Comment by SutoLiu** [comment]
Maven dependencies themselves usually do not cause slow app startups. The issue might be related to dependency conflicts or the initialization code of Firebase Crashlytics.
